hashes/enigma_machine.py

```diff
@@ -1,8 +1,8 @@
 alphabets = [chr(i) for i in range(32, 126)]
-gear_one = [i for i in range(len(alphabets))]
-gear_two = [i for i in range(len(alphabets))]
-gear_three = [i for i in range(len(alphabets))]
-reflector = [i for i in reversed(range(len(alphabets)))]
+gear_one = range(len(alphabets))
+gear_two = range(len(alphabets))
+gear_three = range(len(alphabets))
+reflector = reversed(range(len(alphabets)))
```

Comment:
The old code is constructing a list, but the updated code is giving us a `range` object. Is this ok?

Reply:
no because range returns a list

Reply:
Range returns a `range` object, which does not have the same methods as a list. Now that I realise, yes, I believe this is a problem. I will encase these range objects in the list function